Zend/zend_gc.c

```diff
@@ -463,13 +498,15 @@ static void gc_trace_ref(zend_refcounted *ref) {
 }
 #endif
 
+/* Mark a root buffer entry unused */
 static zend_always_inline void gc_remove_from_roots(gc_root_buffer *root)
 {
 	GC_LINK_UNUSED(root);
 	GC_G(num_roots)--;
 	GC_BENCH_DEC(root_buf_length);
 }
 
+/* Destroy the root buffer */
 static void root_buffer_dtor(zend_gc_globals *gc_globals)
```

Comment:
These two comments seem redundant with the function name

Reply:
On the second, I agree, and have removed the comment. On the first, I'm putting it into different words, which I think is valuable. The name says "remove from roots" and the comment says "mark unused", which are complementary ways of looking at the operation. But I can remove it if you insist.

Reply:
I agree :+1: